metricbeat/module/nats/_meta/run.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+# This script is used with old and new versions of nats,
+# and they use different names for their binaries, detect
+# which one is used and run it.
+
+# NATS 2.X
+if [ -x /opt/nats/nats-server ]; then
+	exec /opt/nats/nats-server -c /opt/nats/nats-server.conf
+fi
+
+# NATS 1.X
+if [ -x /opt/nats/gnatsd ]; then
```

Comment:
Random question: it is not possible to have both binaries in the image, right? Otherwise both will be executed

Reply:
Sorry, there is an `exec`... my bad 🤦‍♀